src/interfaces.py

```diff
@@ -1,60 +1,69 @@
+"""List of defined interfaces."""
 import ipaddress
 import json
 import logging
 
-from ops.framework import EventBase, EventSource, EventsBase, StoredState
-from ops.framework import Object
 from cryptography.hazmat.backends import default_backend
 from cryptography.hazmat.primitives.serialization import load_pem_private_key
 from cryptography.x509 import load_pem_x509_certificate
+from ops.framework import EventBase, EventSource, Object, ObjectEvents, StoredState
```

Comment:
This may violate `Line too long (82 > 79 characters) (E501)`
https://www.flake8rules.com/rules/E501.html
If you install flake8 and run it, you may find that out.

That's probably the reason it will split into two initially.

Reply:
pyproject.toml has the limit set to 99 which is from the template used by other charms.